api_tests/base/test_sloan_study.py

```diff
@@ -83,16 +80,13 @@ def test_sloan_study_variable(self, app, user, preprint):
 
         cookies = resp.headers.getall('Set-Cookie')
 
-        assert f' dwf_{SLOAN_COI_DISPLAY}=True; Domain=localhost; Path=/; samesite=None; Secure' in cookies
-        assert f' dwf_{SLOAN_DATA_DISPLAY}=True; Domain=localhost; Path=/; samesite=None; Secure' in cookies
-        assert f' dwf_{SLOAN_PREREG_DISPLAY}=True; Domain=localhost; Path=/; samesite=None; Secure' in cookies
+        assert f' dwf_{SLOAN_COI_DISPLAY}=True; Domain=localhost; Path=/; samesite=None' in cookies
```

Comment:
https://github.com/CenterForOpenScience/osf.io/pull/9336/files#diff-c1703080e33331cb3e1bfb4f1ddfeb01R311 seems to say that samesite can't be set unless you are over a secure setting. Am I misunderstanding?

Reply:
Oh you can set to what ever you want, the browser will just ignore it, if it's not over https.